cmd/cdi-importer/importer.go

```diff
@@ -135,6 +135,21 @@ func main() {
 		touchDoneFile()
 		preallocationApplied = processor.PreallocationApplied()
 	}
+	// With writeback cache mode it's possible that the process will exit before all writes have been commited to storage.
```

Comment:
Maybe make this a defer so we are ensured it is called. And put it above `if contentType != string(cdiv1.DataVolumeKubeVirt) && (source == controller.SourceRegistry || source == controller.SourceImageio) `

Reply:
no problem, just question - why arent we ensured right now?

Reply:
For instance if there is an error in writing the termination message, we call the exit function, and the defer will get executed before we exit. Right now we won't execute.

Reply:
Actually just double checked, exit doesn't honor defer either.

Reply:
Moved to defer above, thx